internal/pkg/cli/deploy/svc.go

```diff
@@ -538,12 +538,13 @@ type StackRuntimeConfiguration struct {
 	// Use *string for three states (see https://github.com/aws/copilot-cli/pull/3268#discussion_r806060230)
 	// This is mainly to keep the `workload package` behavior backward-compatible, otherwise our old pipeline buildspec would break,
 	// since previously we parsed the env region from a mock ECR URL that we generated from `workload package``.
-	ImageDigest        *string
-	EnvFileARN         string
-	AddonsURL          string
-	RootUserARN        string
-	Tags               map[string]string
-	CustomResourceURLs map[string]string
+	ImageDigest         *string
+	EnvFileARN          string
+	AddonsURL           string
+	RootUserARN         string
+	PermissionsBoundary string
```

Comment:
Should this be part of "runtime" config? 
RuntimeConfig was meant to be about values that get populated during the execution of the `svc deploy` command (like we need to get the S3 template location after uploading the addon template).

In this scenario, do we even need to pass it here? isn't the permission boundary value already included in the `App *config.Application` field that gets passed to the `stack` pkg